templates/docs/examples/patterns/chip/dark.html

```diff
@@ -3,28 +3,29 @@
 
 {% block standalone_css %}patterns_chip{% endblock %}
 
+{% set is_dark = true %}
```

Comment:
I know that this way would be more consistent with other components we are migrating, but we already have the variants example with both light and dark theme, and I think it's OK to leave it as is, so we can also easier compare if the changes work as expected.

So please revert the variants example back to having both light and dark chips, thanks!